src/DependencyInjection/HttplugExtension.php

```diff
@@ -549,7 +549,10 @@ private function configureAutoDiscoveryClients(ContainerBuilder $container, arra
      */
     public function getConfiguration(array $config, ContainerBuilder $container): ?ConfigurationInterface
     {
-        return new Configuration($container->getParameter('kernel.debug'));
+        /** @var bool $debug */
```

Comment:
can you try configuring the phpstan symfony extension as per https://github.com/phpstan/phpstan-symfony?tab=readme-ov-file#configuration ? it should make phpstan aware of the symfony specific things and know the types of container->get and getParameter.

Reply:
Tricky since no `var` folder is generated (only PHPUnit will create those files). I added what I can to the `phpstan.neon.dist`.

Reply:
Also I can't include files like `vendor/phpstan/*` since they will be inside the global composer folder : `~/.composer/vendor/phpstan*`